drawLoop.go

```diff
@@ -39,42 +39,46 @@ func (w *Window) drawLoop() {
 				case <-w.drawCh:
 					break loadingSelect
 				case <-w.animationFrame:
-					draw.Draw(w.winBuffer.RGBA(), w.winBuffer.Bounds(), w.bkgFn(), zeroPoint, draw.Src)
+					w.publish()
+					draw.Draw(w.winBuffers[w.bufferIdx].RGBA(), w.winBuffers[w.bufferIdx].Bounds(), w.bkgFn(), zeroPoint, draw.Src)
 					if w.LoadingR != nil {
-						w.LoadingR.Draw(w.winBuffer.RGBA(), 0, 0)
+						w.LoadingR.Draw(w.winBuffers[w.bufferIdx].RGBA(), 0, 0)
 					}
-					w.publish()
 				case <-w.DrawTicker.C:
-					draw.Draw(w.winBuffer.RGBA(), w.winBuffer.Bounds(), w.bkgFn(), zeroPoint, draw.Src)
+					w.publish()
+					draw.Draw(w.winBuffers[w.bufferIdx].RGBA(), w.winBuffers[w.bufferIdx].Bounds(), w.bkgFn(), zeroPoint, draw.Src)
 					if w.LoadingR != nil {
-						w.LoadingR.Draw(w.winBuffer.RGBA(), 0, 0)
+						w.LoadingR.Draw(w.winBuffers[w.bufferIdx].RGBA(), 0, 0)
 					}
-					w.publish()
 				}
 			}
 		case f := <-w.betweenDrawCh:
 			f()
 		case <-w.animationFrame:
-			draw.Draw(w.winBuffer.RGBA(), w.winBuffer.Bounds(), w.bkgFn(), zeroPoint, draw.Src)
+			w.publish()
+			draw.Draw(w.winBuffers[w.bufferIdx].RGBA(), w.winBuffers[w.bufferIdx].Bounds(), w.bkgFn(), zeroPoint, draw.Src)
 			w.DrawStack.PreDraw()
 			p := w.viewPos
-			w.DrawStack.DrawToScreen(w.winBuffer.RGBA(), &p, w.ScreenWidth, w.ScreenHeight)
-			w.publish()
+			w.DrawStack.DrawToScreen(w.winBuffers[w.bufferIdx].RGBA(), &p, w.ScreenWidth, w.ScreenHeight)
 		case <-w.DrawTicker.C:
-			draw.Draw(w.winBuffer.RGBA(), w.winBuffer.Bounds(), w.bkgFn(), zeroPoint, draw.Src)
+			// Theory: publish what was drawn last frame to screen, then work on preparing the next frame.
```

Comment:
While this comment made sense during the testing now that we are commiting if I were a user it would be perhaps disconcerting to see this comment leading with the word 'Theory:'

Long story short I think it would be great to do 
```suggestion
			// Publish what was drawn last frame to screen, then work on preparing the next frame.
```